testRunner.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&display=swap" rel="stylesheet">
+
+  <link rel="apple-touch-icon" sizes="180x180" href="public/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="public/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="public/favicon-16x16.png">
+  <link rel="manifest" href="public/site.webmanifest">
+
+  <title>News feed Test Suite</title>
+
+  <link rel="stylesheet" href='public/css/testRunner.css'>
+  
+  <script src="src/newsList.js" defer></script>
+  <script src="src/newsItem.js" defer></script>
+  <script src="src/interface.js" defer></script>
+  <script src="lib/testinglib.js" defer></script>
+  <script src="spec/newsItemSpec.js" defer></script>
+  <script src="spec/newsListSpec.js" defer></script>
+  <script src="spec/interfaceSpec.js" defer></script>
+
+</head>
+<body>
+
+<div class='resultBox'>
+  <H1>News Feed Test Suite</H1>
+</div>
+  
+  <div class="row">
+    <div class="column">
+      <div id='testResults'></div>
+    </div>
+    <div class="column">
+      <div id="indexDisplay"></div>
+    </div>
+  </div>
+  
+</body>
+</html>
```

Comment:
I like how you set up your testing suite, and how it displays the HTML along side it. 

Would have been simpler for me to do the same instead of trying to tear it down after each test!